### PR TITLE
[MNG-5600] Dependency management import should support exclusions

### PR DESCRIPTION
**Pull request by alex1989hu**
The original patch is made by: Christian Schulte <schulte@apache.org>
@ChristianSchulte

For more information kindly ask you to visit the original pull request  #30

### CODE REVIEW

File: core-it-suite/src/test/resources/mng-5600/settings-template.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rfscholte) [inline core-it-suite/src/test/resources/mng-5600/settings-template.xml:1]: I don't understand the need for this file

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5600DependencyManagementImportExclusionsTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ *
+ * @author Christian Schulte
+ */
+public class MavenITmng5600DependencyManagementImportExclusionsTest
+    extends AbstractMavenIntegrationTestCase
+{
+
+    public MavenITmng5600DependencyManagementImportExclusionsTest()
+    {
+        super( "[3.4.0,)" );
```
[the remark this answers is not in the document]
Comment (alex1989hu) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5600DependencyManagementImportExclusionsTest.java]: Replaced to your proposal

File: core-it-suite/src/test/resources/mng-5600/settings-template.xml (hunk 1 above)
Comment (alex1989hu) [reply core-it-suite/src/test/resources/mng-5600/settings-template.xml:1]: The original author does not respond, so try to figure out. The repository contains 201 XML file with the same name but with different content. If you are familiar with IT tests in Maven you could tell is it needed or not. I guess the author wanted to follow convention.

File: core-it-suite/src/test/resources/mng-5600/exclusions/pom.xml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.apache.maven.its.mng5600</groupId>
+        <artifactId>bom</artifactId>
+        <version>0</version>
+        <type>pom</type>
+        <scope>import</scope>
+        <exclusions>
```
[the remark this answers is not in the document]
Comment (alex1989hu) [reply core-it-suite/src/test/resources/mng-5600/exclusions/pom.xml:44]: I don't get you. If you exclude a dependency, then the actual excluded dependency and its all transitive dependencies are excluded. Maybe I misinterpret the change. Please clarify it

[the remark this answers is not in the document]
Comment (mistriel) [reply core-it-suite/src/test/resources/mng-5600/exclusions/pom.xml:44]: @rfscholte  that is exactly the case. Sometimes you import another pom that is itself a `dependencyManagement` like `spring-boot-dependencies`

**Comment by gnodet** [comment]
Superseded by #269

**Pull request closed without merging**